Release-Checklist.md

```diff
@@ -0,0 +1,79 @@
+Change release process, instead create a release candidate we can test the HEAD what will be the last commit using new automated obs package generation we can generate and test commits based on HEAD.
+
+- [ ] Raise PRs to update the `VERSION` file in the following repositories:
+   - [ ] [agent][agent].
+   - [ ] [proxy][proxy].
+   - [ ] [runtime][runtime].
+   - [ ] [shim][shim].
+   - [ ] [throttler][throttler].
+
+  Note that the "phase" element of the project encoded in the version strings needs to match for all components. For example, when a `beta` is released, the version string for *all* components should show `beta`.
+
+- [ ] Ensure all CI tests pass **for all architectures**.
```

Comment:
This ideally would be tested in the CI when we update versions file on a PR. We can add a section with the sub task that the CI should be able to done ( and add it if has not support today).

Reply:
I think we should be specific on which architectures this will be done.  Currently this isn't really a list AFAIU...

Reply:
Well, since the CI currently only supports x86_64, do we want to just list that? @bergwolf - any thoughts?